Update documentation with textual improvements (#1170)
asreview/asreview: docs/source/contribute.rst

Support
-------

> Questions can be asked on `GitHub Discussions <https://github.com/asreview/asreview/discussions>`__. 
> 
> For bug reports and feature requests, please submit an issue on `GitHub <https://github.com/asreview/asreview/issues/new/choose>`__.
> 

Reviewer: ```suggestion
Questions can be asked on `GitHub Discussions <https://github.com/asreview/asreview/discussions>`__. For bug reports and feature requests, please submit an issue on `GitHub <https://github.com/asreview/asreview/issues/new/choose>`__.

```

Contribute
~~~~~~~~~~

> How do you go from user to contributor? There are many ways to join in, and it might be less complicated than you expect. In a `blogpost <https://asreview.nl/blog/open-source-and-research/>`_ we list some easy examples for first-time contributors, for example sharing your experiences or answering users questions on the `Dicsussion platform <https://github.com/asreview/asreview/discussions>`_. 

Reviewer: ```suggestion
How do you go from user to contributor? There are many ways to join in, and it might be less complicated than you expect. In a `blogpost <https://asreview.nl/blog/open-source-and-research/>`_, we list some easy examples for first-time contributors, for example sharing your experiences or answering user questions on the `Discussion platform <https://github.com/asreview/asreview/discussions>`_. 
```

Specific instructions for code-contributing are available on `Github <https://github.com/asreview/asreview/blob/master/CONTRIBUTING.md>`_ as well as instructions for `developers <https://github.com/asreview/asreview/blob/master/DEVELOPMENT.md>`_.

.. note::

> 	All contributions, small or large, are very much appreciated!!

Reviewer: ```suggestion
	All contributions, small or large, are very much appreciated!
```
